src/App.js

```diff
@@ -0,0 +1,37 @@
+import React, {useState, useEffect} from "react";
+import "./App.css";
+import Player from "./Player";
+import {db} from "./firebase";
+import 'moment-timezone';
```

Comment:
why are we importing this here? It doesn't look like  we're using `moment` at all in this file

Reply:
Resolved